src/components/Dropdown/Dropdown.tsx

```diff
@@ -141,7 +164,7 @@ export const Dropdown: FC<DropdownProps> = ({
                 </button>
             </Trigger>
             <AnimatePresence>
-                {!disabled && isOpen && (
+                {!disabled && isOpen && maxHeight !== DEFAULT_DROPDOWN_MAX_HEIGHT && (
```

Comment:
Why is it needed?

Reply:
It basically prevents the dropdown from rendering and then the max height changing immediately afterwards causing a small visual bug